pkg/logging/http.go

```diff
@@ -39,7 +39,11 @@ func (m *HTTPServerMiddleware) HTTPMiddleware(name string, next http.Handler) ht
 	return func(w http.ResponseWriter, r *http.Request) {
 		wrapped := httputil.WrapResponseWriterWithStatus(w)
 		start := time.Now()
-		_, port, err := net.SplitHostPort(r.Host)
+		hostPort := r.Host
```

Comment:
What should we do, if both `r.Host` and `r.URL.Host` is set, which one should we display?

Reply:
If `r.Host` is there, it will get priority. We are only using `r.URL.Host` if `r.Host` is an empty string.

Reply:
I think we can print anyone of them, as long one of them is non-empty.

Reply:
From the docs:

```
    // For client requests, the URL's Host specifies the server to
    // connect to, while the Request's Host field optionally
    // specifies the Host header value to send in the HTTP
    // request.
```

So, we should give priority to `r.Host` over `r.URL.Host` because `r.Host` _might_ be more specific i.e. it's the `Host` header :stuck_out_tongue:

Reply:
Looks solved then 😄 !

Reply:
👍🏽